fabric/fabric-commands/src/main/java/io/fabric8/commands/ProfileEditAction.java

```diff
@@ -215,7 +217,16 @@ private void editProfile(Profile profile) throws Exception {
         }
 
         if (pidProperties != null && pidProperties.length > 0) {
-            editInLine = handlePid(builder, pidProperties, profile);
+            
+            if(validateProfileEditInput(pidProperties) || (delete || remove)){
```

Comment:
Also, the code style can be cleaner: There should be spaces surrounding `if` parentheses.